### DISCUSSION TITLE
coding/bot

### DISCUSSION THREAD

**Discussion by giscus** [lxl66566/lxl66566.github.io#17] [General]
# coding/bot

bot

https://absx.pages.dev/coding/bot.html

**Comment by yaowan233** [comment]
急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了急了
nonebot2你也可以用适配器适配多平台，tg，discord，kook，minecraft，b站都有适配，现在nonebot2也有图形化操作了，文档也改了

**Reply by lxl66566** [reply]
好好，我改
然而已经对做 bot 这件事本身失去兴趣了（
